src/main/webapp/main.js

```diff
@@ -30,3 +32,47 @@ Promise.all([mapPromise, domPromise]).then(() => {
 function initMap() {
   myMap = new ChapMap();
 }
+
+
+connection.onopen = () => {
+};
+
+connection.onclose = () => {
+};
+
+connection.onerror = (error) => {
+  console.error(error);
+};
+
+connection.onmessage = (event) => {
+  let li = document.createElement('li');
+  
+  var obj = JSON.parse(event.data); 
+
+  if(obj.type !== 'MSG_RECV'){
```

Comment:
I would change this if statement to a switch statement. There will be multiple object types (MSG_RECV, MSG_SEND, MAP_RECV, MAP_SEND) and possibly more.